config/config.go

```diff
@@ -66,6 +66,25 @@ func LoadConfig(locations ...string) (*Config, error) {
 	return conf, nil
 }
 
+// Merge method is used to override current config with new one
+func (c *Config) Merge(cfg Config) {
+	if cfg.Endpoint != "" {
+		c.Endpoint = cfg.Endpoint
+	}
+
+	if cfg.Insecure {
```

Comment:
What about situation when current config has `c.Insecure = True` and new config has `cfg.Insecure = False`?